lib/vsc/administration/user.py

```diff
@@ -402,13 +413,12 @@ def __setattr__(self, name, value):
         super(VscTier2AccountpageUser, self).__setattr__(name, value)
 
 
-
 cluster_user_pickle_location_map = {
     'kyukon': VscTier2AccountpageUser,
 }
 
 cluster_user_pickle_store_map = {
```

Comment:
same as above

Reply:
This needs to be here as `VscTier2AccountpageUser` first needs to be defined.